staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr_test.go

```diff
@@ -47,7 +72,7 @@ func TestIntOrStringUnmarshalJSON(t *testing.T) {
 		input  string
 		result IntOrString
 	}{
-		{"{\"val\": 123}", FromInt(123)},
```

Comment:
I guess in a few of these tests you could have kept both methods, we still need to test the other one. That's a nit, I don't think it matters at all.

Reply:
I left the main `TestFromInt` test; IMO that’s what determines that `FromInt` meets its spec. The rest of the tests use the recommended function which is now `FromInt32`.